backend/api/models/computeplan.py

```diff
@@ -89,7 +87,7 @@ def get_task_stats(self) -> dict:
             waiting_executor_slot_count=Count(
                 "key", filter=Q(status=ComputeTask.Status.STATUS_WAITING_FOR_EXECUTOR_SLOT)
             ),
-            doing_count=Count("key", filter=Q(status=ComputeTask.Status.STATUS_DOING)),
+            executing_count=Count("key", filter=Q(status=ComputeTask.Status.STATUS_EXECUTING)),
```

Comment:
This is unused if you don't reactivate the case on `self.Status.PLAN_STATUS_TODO`

Reply:
It is used in frontend https://github.com/Substra/substra-frontend/pull/299/files#diff-a70efa5998c05ae5dfe367770344a8a5598e6e38c1d27a340bac6cd546cc38c6R8-R9